### PR TITLE
Update the query tutorial

### PR DESCRIPTION
**Pull request by ironage**
Updating the query documentation to give more examples of backlinks and string timestamps.

### CODE REVIEW

File: docs/tutorials/query-language.md [hunk 1]
```diff
@@ -98,18 +109,46 @@ let teens = realm.objects('Contact').filtered('SUBQUERY(friends, $friend, $frien
...
...rtain type of car? We can find this using backlinks. Backlink is a term used to describe following a relationship backwards. This backwards relationship always exists for every link property but it is not required to name it in the model. We can use the named backlink (example 2 below) just like we did with a forward relationship. However, you can also query backlinks without a name by fully describing the forward relationship using the ` @links.ClassName.PropertyName` syntax (example 3 below). Regardless of whether they are named or not, backlinks can be treated like a collection. This means that you can use all the collection operators as usual (examples 4-7 below). In addition to the collection operators, there is special syntax to allow querying the count of all backlink relationships (example 8 below) which is different than querying for the backlink count on a specific relationship (example 5 below) unless there is only one incoming link in total. Although there is no functional difference between using named versus unnamed backlinks, the readability of the query syntax is affected so it is normally preferred to name the backlink with a `linkingObject` property when possible.
```
Comment (kneth) [inline docs/tutorials/query-language.md]: Maybe: 
"property but it is not required" -> "property and it is not required"

Comment (kneth) [reply docs/tutorials/query-language.md]: You mention "example 5" (and others), and cal it "Query 5" in the example. Maybe it should be "query 5 in the example below".

**Comment by ironage** [comment]
@kneth do you know why https://realm.io/docs/javascript/latest/api/tutorial-query-language.html hasn't updated?

**Comment by kneth** [comment]
@ironage It looks like the release of 2.14.0 didn't update the documentation. I will update the website at next release.

**Pull request merged**